### PR TITLE
remove destinationCode from packing slip

### PR DESCRIPTION
**Pull request by jarrilla**
assume that it's in items array from FE.

~no changes to pipeline necessary.
All items are being sent to FE correctly.~

### CODE REVIEW

**Comment by jarrilla** [comment]
Minor pipeline changes were indeed necessary. had to match packing slips to their (work order item, router-step) pair

File: src/workOrder/controller.js [hunk 1]
```diff
@@ -241,6 +251,8 @@ async function getAllWithPackedQties(showFulfilled) {
     const d = await ShopQueue.aggregate(agg);
     const data = d?.[0]?.activeWorkOrders;
 
+    console.debug(data.filter(x => x.orderNumber === 'WAABI1006'));
```
Comment (joshplumley) [inline src/workOrder/controller.js:254]: @jarrilla did you mean to leave this in here?

**Review by joshplumley — APPROVED** [review]
This all seems to work. Just a question about the console.debug

File: src/workOrder/controller.js (hunk 1 above)
Comment (jarrilla) [reply src/workOrder/controller.js:254]: Nope. Accidentally left it in there after i was using that order number as a good sample for bad data. Thanks for catching it

**Pull request merged**